doc/WEATHER_TYPE.md

```diff
@@ -20,46 +19,27 @@ Each weather type is a type of weather that occurs, its effects and what causes
 | `rains`                        | Whether said precipitation falls as rain.                             |
 | `acidic`                       | Whether said precipitation is acidic.                                 |
 | `tiles_animation`              | Optional, name of the tiles animation to use                          |
-| `sound_category`               | Optional, if playing sound effects what to use. Valid values are: silent, drizzle, rainy, thunder, flurries,
-    snowstorm and snow. |
+| `sound_category`               | Optional, if playing sound effects what to use. Valid values are: silent, drizzle, rainy, thunder, flurries, snowstorm and snow. |
```

Comment:
A bit unclear what is meant by "if playing sound effects what to use". "what sound effect to use by the sound system"?

Reply:
"what sound effect to play"?